metadata/registry.go

```diff
@@ -1,11 +1,12 @@
 package metadata
 
 import (
-	"encoding/json"
 	"fmt"
 	"os"
 	"path/filepath"
 
+	yaml "gopkg.in/yaml.v2"
```

Comment:
Any reason we're using this package here instead of using what the other places are using: `"github.com/ghodss/yaml"`?

Reply:
Arg.